Proxy/HTTP/IncorrectContentLength.bambda

```diff
@@ -4,7 +4,7 @@
  * @author albinowax
  **/
 
-if (!requestResponse.hasResponse()) {
+if (!requestResponse.hasResponse() || requestResponse.request().method() == "HEAD") {
```

Comment:
Good suggestion! Can you please update this to:

requestResponse.request().method().equals("HEAD")

as the == checks reference equality.